packages/calcite-components/src/utils/iconScaleAdjuster.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { getIconScale } from "./iconScaleAdjuster";
```

Comment:
This file is no longer needed.

Reply:
Hi @jcfranco removed the file, kindly check. Thanks